vignettes/r2dii-analysis.Rmd

```diff
@@ -12,117 +12,121 @@ knitr::opts_chunk$set(
 
-``` {r r2dii libraries}
+``` {r use-r2dii}
 library(r2dii.data)
 library(r2dii.match)
 library(r2dii.analysis)
 ```
 
-We also recommend the following two `tidyverse` packages to manipulate and plot data:
+We also recommend packages in the [tidyverse](https://www.tidyverse.org/); they are optional but useful.
 
-``` {r tidyverse libraries}
-library(dplyr, warn.conflicts = FALSE)
-library(ggplot2)
+``` {r use-tidyverse}
+library(tidyverse)
 ```
 
 ## Match your loanbook to climate-related asset-level data
 
 See [r2dii.match](https://2degreesinvesting.github.io/r2dii.match) for a more complete description of this process. 
 
 ```{r match}
-matched <- match_name(
-  loanbook = loanbook_demo,
-  ald = ald_demo,
-  overwrite = overwrite_demo
-) %>%
-  prioritize()
+# Use these datasets to practice but eventually you should use your own data.
+# The optional syntax `package::data` is to clarify where the data comes from.
+loanbook <- r2dii.data::loanbook_demo
+ald <- r2dii.data::ald_demo
+
+matched <- match_name(loanbook, ald) %>% prioritize()
 
 matched
 ```
 
 ## Calculate targets
 
-You can calculate scenario targets using two different approaches: Market Share Approach, or Sectoral Decarbonization Approach
+You can calculate scenario targets using two different approaches: Market Share Approach, or Sectoral Decarbonization Approach.
 
 ### Market Share Approach
 
-The [Market Share Approach](https://2degreesinvesting.github.io/r2dii.analysis/articles/target-sda.html) is used to calculate scenario targets for the `production` of a technology in a sector. For example, we can use this approach to set targets for the production of electric vehicles in the automotive sector. This approach is recommended for sectors where a granular technology scenario roadmap exists. 
+The [Market Share Approach](https://2degreesinvesting.github.io/r2dii.analysis/articles/target-sda.html) is used to calculate scenario targets for the `production` of a technology in a sector. For example, we can use this approach to set targets for the production of electric vehicles in the automotive sector. This approach is recommended for sectors where a granular technology scenario roadmap exists.
 
 Targets can be set at the portfolio level:
 
-```{r portfolio}
+```{r market-share-targets-portfolio}
+# Use these datasets to practice but eventually you should use your own data.
+scenario <- r2dii.data::scenario_demo_2020
+regions <- r2dii.data::region_isos_demo
+
 market_share_targets_portfolio <- matched %>%
   target_market_share(
-    ald = ald_demo,
-    scenario = scenario_demo_2020,
-    region_isos = region_isos_demo
+    ald = ald,
+    scenario = scenario,
+    region_isos = regions
```

Comment:
This is to make the code easier to work when users use their own data (see my previous comment).